api/registrations/serializers.py

```diff
@@ -314,7 +314,6 @@ def get_view_only_links_count(self, obj):
         return obj.private_links.filter(is_deleted=False).count()
 
     def update(self, registration, validated_data):
-        # TODO - when withdrawal is added, make sure to restrict to admin only here
```

Comment:
This hasn't been implemented - write contributors can still incorrectly withdraw registrations.  The Permission class on the RegistrationDetail view is `ContributorOrPublic`, so I think the intent is to allow write contribs to make some edits (like adding tags).  That means we need to enforce admin-only individually, here.